src/ontology/edits/oeo-physical.omn

```diff
@@ -9262,6 +9262,121 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1359",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1256
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1362",
+        rdfs:label "heat recovery steam generator"
+    
+    SubClassOf: 
+        OEO_00140102,
+        OEO_00000503 some OEO_00110001,
+        OEO_00010234 some OEO_00000007
+    
+    
+Class: OEO_00310025
+
+    Annotations: 
+        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Combined cycle electricity generation is a fuel-powered electricity generation process in which a gas turbine process is followed by an steam-electric process.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1256
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1362",
+        rdfs:label "combined cycle electricity generation"
+    
+    SubClassOf: 
+        OEO_00050001,
+        OEO_00010234 some OEO_00000139,
+        OEO_00010234 some OEO_00240000,
+        <http://purl.obolibrary.org/obo/RO_0002427> some OEO_00050020,
+        <http://purl.obolibrary.org/obo/RO_0002427> some OEO_00310027
+    
+    
+Class: OEO_00310027
+
+    Annotations: 
+        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A gas turbine process is an energy transformation that converts chemical energy into rotational energy using a continous internal combustion process. A gas turbine is participating in a gas turbine process.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1256
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1362",
+        rdfs:label "gas turbine process"
+    
+    SubClassOf: 
+        OEO_00020003,
+        OEO_00000532 some OEO_00000099,
+        OEO_00010234 some OEO_00000007,
+        OEO_00010235 some OEO_00240000,
+        <http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00140038,
+        <http://purl.obolibrary.org/obo/RO_0000057> some OEO_00000185
+    
+    
+Class: OEO_00310028
+
+    Annotations: 
+        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A steam power unit is a power generating unit using steam.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1256
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1362",
+        rdfs:label "steam power unit"
+    
+    SubClassOf: 
+        OEO_00000334,
+        OEO_00000503 some OEO_00110001,
+        OEO_00010234 some OEO_00000007,
```

Comment:
What is a steam turbine using chemical energy for?

Reply:
I will also delete this axiom.

Reply:
In issue #1256 I see the proposal on the definition, but no discussion and hence no consensus on the definition.

To me, this definition is not sufficient as as it is not clear. Do we mean here, that a `steam power unit` is _every_ power generating unit? Then it should be implemented as equivalent class.

Or do we mean a power generating unit, that has has only a steam turbine but no other turbines or motors? Then we should mention that in the definition (and axiomtise that).